setup.py

```diff
@@ -4,15 +4,15 @@
 long_description = (Path(__file__).parent / "README.md").read_text()
 
```

Comment:
S (Substantive): IMO we should convert this to a Habushu project to make it more consistent w/ our other builds and use a `pyproject.toml` file.  This will allow publishing to PyPI more easily.